contracts/BadgerVotingShare.sol

```diff
@@ -32,21 +35,42 @@ contract BadgerVotingShare {
     ISett constant sett_badger_wBTC_SLP =
         ISett(0x1862A18181346EBd9EdAf800804f89190DeF24a5);
     IGeyser constant geyser_badger_wBTC_SLP =
-        IGeyser(0xB5b654efBA23596Ed49FAdE44F7e67E23D6712e7); 
+        IGeyser(0xB5b654efBA23596Ed49FAdE44F7e67E23D6712e7);
 
     // Rari pool - fBADGER-22
     ICToken constant fBADGER =
         ICToken(0x6780B4681aa8efE530d075897B3a4ff6cA5ed807);
 
-    IBridgePool constant aBADGER = IBridgePool(0x43298F9f91a4545dF64748e78a2c777c580573d6);
+    IBridgePool constant aBADGER =
+        IBridgePool(0x43298F9f91a4545dF64748e78a2c777c580573d6);
 
     ICurvePool constant badger_wBTC_crv_pool =
         ICurvePool(0x50f3752289e1456BfA505afd37B241bca23e685d);
-    ICurveToken constant badger_wBTC_crv_token = 
+    ICurveToken constant badger_wBTC_crv_token =
         ICurveToken(0x137469B55D1f15651BA46A89D0588e97dD0B6562);
     ISett constant sett_badger_wBTC_crv =
         ISett(0xeC1c717A3b02582A4Aa2275260C583095536b613);
 
+    // Balancer Vault
+    IVault constant balancer_vault =
+        IVault(0xBA12222222228d8Ba445958a75a0704d566BF2C8);
+
+    IBalancerPoolToken constant badger_wBTC_balancer =
+        IBalancerPoolToken(0xb460DAa847c45f1C4a41cb05BFB3b51c92e41B36);
+    ISett constant sett_badger_wBTC_balancer =
+        ISett(0x63ad745506BD6a3E57F764409A47ed004BEc40b1);
+    IERC20 constant bptStakedBadgerWbtc =
+        IERC20(0x3F29e69955E5202759208DD0C5E0BA55ff934814);
+    IERC20 constant bptAuraBadgerWbtc =
+        IERC20(0xf9Ad7c6BAB11d9F483Fd40eE4170bf27F21D4C78);
```

Comment:
should not be this addr be instead the reward contract and read the `balanceOf` directly from there?

ref: [reward contract](https://etherscan.io/address/0xddf14a569dd91af895e3b05d6dbcbb9db1c3834c)

Reply:
yes you are right, holders of the aura wrapper itself are an edge case, the staked position thereof is what matters.

Reply:
9cd30aa1d439c61c506630aa238193d7e9fd93bc